config/locales/es.yml

```diff
@@ -1826,8 +1830,7 @@ es:
           title: Favor de compartir la dirección de su correo electrónico
         file_full_return:
           full_btn: Presente su declaración de impuestos completa
-          help_text:
-          - 'Con la presentación de una declaración de impuestos completa, podría cobrar todo el dinero que se le debe. Considere esta opción si:'
+          help_text: 'Con la presentación de una declaración de impuestos completa de %{current_tax_year}, podría cobrar todo el dinero que se le debe. Considere esta opción si:'
```

Comment:
This was an amusing/disturbing situation where the Spanish content didn't state a year, but the English did. Ho ho ho.